modules/getting-started-cli-deploying-python-app.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-cli.adoc
+
+[id="getting-started-cli-deploying-python-app_{context}"]
+= Deploying a Python application
+
+The following procedure deploys a backend service for the `parksmap` application. The backend service provides data on major national parks across the world.
+
+The deployed backend service that is `nationalpark-katacoda`. The service is a Python application that returns map coordinates of major national parks as JSON via a REST service API.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+* You have a deployed image.
+
+.Procedure
+
+. Create new Python application, enter the following command:
++
+[source,terminal]
+----
+$ oc new-app https://github.com/openshift-roadshow/nationalparks-katacoda
```

Comment:
This is 5 years old, we updated this here:

Repo: https://github.com/openshift-roadshow/nationalparks-py

Reply:
@blues-man This doesn't work. It doesn't populate the map like it should.

Reply:
@blues-man I can't get the new link you provided above to work? It doesn't display the parks like it is supposed to and I could not use the Java version either as in the example. Are we supposed to be adding a database to this?

Reply:
Yes this need a MongoDB database

Reply:
https://github.com/openshift-labs/starter-guides/blob/ocp-4.8/workshop/content/nationalparks-python.adoc

Reply:
@blues-man Thank you! I'll get this updated :)

Reply:
Awesome! thanks :)